database/db.go

```diff
@@ -540,8 +540,11 @@ func (db *DB) NamedBulkExec(
 // Takes in up to the number of entities specified in count from the arg stream and
 // executes a new transaction that runs a new query for each entity in this set of arguments,
 // until the arg stream has been processed.
+//
 // The transactions are executed in a separate goroutine with a weighting of 1
 // and can be executed concurrently to the extent allowed by the semaphore passed in sem.
+//
+// Note that committing the transaction may not honor the context provided, as described further in DB.ExecTx.
```

Comment:
Please add square brackets to `DB.ExecTx`, i.e. `[DB.ExecTx]` so that it is transformed to a [link](https://tip.golang.org/doc/comment#links) in the documentation.

Reply:
Always these new-fashioned things, <https://github.com/golang/proposal/blob/master/design/51082-godocfmt.md#new-syntax-for-url-links>.